packages/js/msgpack/src/__tests__/msgpack.spec.ts

```diff
@@ -26,8 +26,17 @@ describe("msgpack", () => {
     customMap.set("secondKey", "secondValue")
 
     const encoded = msgpackEncode(customMap)
-    // [199, 43, 1] are being added because of the map structure
-    expect(encoded).toEqual(Uint8Array.from([ 199, 43, 1, ...expectedArrayLike]))
+    const decoded = msgpackDecode(encoded)
+    expect(decoded).toEqual(customMap);
+  });
+
+  it("Should encode and decode nested map, returning the same map", () => {
```

Comment:
I think `returning the same map` is implied

Reply:
Agree, I just kept it as it is but renaming these test name would be better